grails-app/views/layouts/mobile.gsp

```diff
@@ -59,11 +60,16 @@
             withCredentials: true
         },
         beforeSend: function (xhr) {
-            xhr.setRequestHeader('authKey', "${authKey}");
-            xhr.setRequestHeader('userName', "${userName}");
+            <g:if test="${authorization}">
+                xhr.setRequestHeader('Authorization', "${authorization}");
+            </g:if>
+            <g:elseif test="${grailsApplication.config.getProperty("mobile.authKeyEnabled", Boolean) && authKey && userName}">
+                xhr.setRequestHeader('authKey', "${authKey}");
```

Comment:
I am not sure this will work with Cognito

Reply:
API will be disabled before moving to cognito.